app/services/remote_tagging_service.rb

```diff
@@ -0,0 +1,70 @@
+require 'faraday'
+class RemoteTaggingService
+  def initialize(options)
+    @app_name = options[:app_name]
+    @object_type = options[:object_type]
+    @object_id = options[:object_id]
+  end
+
+  def process(action, tag)
+    post_request(action_url(action), tag)
```

Comment:
Will you support deleting tag in the near future? Will it still use POST but not DELETE?

Reply:
@bzwei The add is a temporary solution till we can get the full create/delete tags from the common gem. The API definition might change so once that happens I will add the delete.